### DISCUSSION TITLE
How to disable context menu in texteditor

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1220] [General]
How do I disable context menu in text editor??

**Comment by kkoreilly** [comment]
You can do `w.ContextMenus = nil`; see https://www.cogentcore.org/core/widgets/containers/context-menus.

**Reply by heejit** [reply]
I have set following still showing menu with clear option
`this.TextEditor.Scene.ContextMenus = nil`
![image](https://github.com/user-attachments/assets/99382012-c0dc-4b96-9ca6-d0377504939f)

**Reply by kkoreilly** [reply]
Yes, you need to do `this.TextEditor.ContextMenus = nil` in addition to the `Scene` one to get rid of all of them.

**Reply by heejit** [reply]
thanks it is okay now.
